docs/source/data/kedro_dataset_factories.md

```diff
@@ -223,19 +223,14 @@ The matches are ranked according to the following criteria:
 You can use dataset factories to define a catch-all pattern which will overwrite the default [`MemoryDataset`](/api/kedro.io.MemoryDataset) creation.
 
 ```yaml
-"{a_default_dataset}":
+"{default_dataset}":
```

Comment:
Cannot comment on the target line as it wasn't changed but we need to update the `filepath` as well

Reply:
Good catch! Updated!